tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "target": "es5",
```

Comment:
my editor is giving an error with this removed, lets upgrade it to ESNext insteaad
```suggestion
    "target": "ESNext",
```

Reply:
related to my prettier comment on the packagejson: this file should get a prettier:fix run on it